app/src/main/kotlin/com/wire/android/ui/home/conversations/model/MessageViewWrapper.kt

```diff
@@ -89,7 +89,7 @@ data class MessageBody(
 
 data class User(
     val avatarAsset: UserAvatarAsset?,
-    val availabilityStatus: UserStatus,
+    val availabilityStatus: UserAvailabilityStatus,
```

Comment:
Maybe we should substitute these 2 fields for the `AvatarData` / `SelfUserData` discussed above? just to keep things consistent